CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+Welcome to Espanso!
+We are very happy to have you and we thank you for considering contributing!
+
+We would like to order the contributions like this:
+
+- Before you start hacking, it is important to **ask first** if your idea or bugfix is in order.
+  Create an issue or come and say hi in the `#dev` channel for fixing bugs and adding features involved with coding logic, and `#documentation` for updating primarly the website. Join us at [the discord][`espanso` discord].
+  We don't bite!.
+
+  It would be sad that you do the effort to clone the project, successfully make the PR, but it wasn't in our plans or there is another PR that is currently adressing that issue.
```

Comment:
```suggestion
  It would be sad that you do the effort to clone the project, successfully make the PR, but it wasn't in our plans or there is another PR that is currently addressing that issue.
```